lib/base-wizard.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
```

Comment:
I wonder if we need a base wizard? I just think it might be a little bit of indirection that's not needed. Totes onboard with base error and base controllers though...